src/adapter/src/catalog/builtin.rs

```diff
@@ -2140,6 +2140,24 @@ FROM mz_internal.mz_dataflow_operator_dataflows_per_worker
 WHERE worker_id = 0",
 };
 
+pub const MZ_OBJECT_DEPENDENCIES_TRANSITIVE: BuiltinView = BuiltinView {
+    name: "mz_object_dependencies_transitive",
```

Comment:
`mz_transitive_object_dependencies` as a name may be slightly more grammatically correct? I'm not really sure though.

Reply:
I went for `mz_object_transitive_dependencies` so this table and `mz_object_dependencies` appear close to each other when listed in alphabetical order.